### DISCUSSION TITLE
FAILED. 22 - Find dlinfo failed

### DISCUSSION THREAD

**Discussion by 4xi0ng** [bytedance/android-inline-hook#17] [Q&A]
shadowhook: hook_sym_addr(0x7725eebbac, 0x77c5f42cb8) ...
switch: get dlinfo info: target_addr 7725eebbac, sym_name (NULL), sym_sz 0, load_bias 7725ed2000, pathname /data/local/tmp/libex.so
shadowhook: hook_sym_addr(0x7725eebbac, 0x77c5f42cb8) FAILED. 22 - Find dlinfo failed
怎么回事呢，是我使用方法不对吗

**Comment by caikelun** [comment]
`FAILED. 22 - Find dlinfo failed` 说明这个地址找不到符号信息。如果确认要 hook 没有符号信息的函数，可以用 `shadowhook_hook_func_addr` 试试。
